ChefExtensionHandler/bin/chef-install.psm1

```diff
@@ -72,7 +74,21 @@ function Install-ChefClient {
   $env:Path += ";C:\\opscode\\chef\\bin;C:\\opscode\\chef\\embedded\\bin"
 
   Install-AzureChefExtensionGem $chefExtensionRoot
+}
 
+function Download-ChefClient {
+  $remoteUrl = "http://www.chef.io/chef/download?p=windows&pv=2012&m=x86_64&v=latest&prerelease=false"
```

Comment:
@siddheshwar-more  I don't think there is the need for this since its going to download the same chef-client for different windows platform. Look at https://downloads.chef.io/chef-client/windows/. @adamedx Please make me correct if I am wrong here .